### ISSUE TITLE
Remove from Cart (X) does not remove from cart.

### ISSUE THREAD

**Issue by machikoyasuda** [reactioncommerce/reaction#3669]
### Expected behavior
- Clicking on the (X) on an item in the cart should remove it from the cart.

### Actual behavior
- Clicking on the (X) on an item in the cart does not remove it. 
- No console errors

Result: There is _no_ way for a user to remove an item from the cart.

### Steps to reproduce the behavior
1. Add item to cart
2. Click cart
3. Click the X -- nothing happens

### Versions

```
Node: 8.2.1
NPM: 5.6.0
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.27.0
Reaction: 1.8.0
Reaction branch: release-1.8.0
Docker: 17.09.0-ce
```

Browser: http://whatismybrowser.com/w/KE449ZE

### QA

QA by @willopez + @machikoyasuda

**Comment by zenweasel** [comment]
This is a duplicate

**Comment by zenweasel** [comment]
Sorry, my mistake. Not a dupe. Was a PR but no issue.

**Comment by zenweasel** [comment]
Closed via #3740
